ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/StandaloneApiServer.scala

```diff
@@ -69,33 +68,28 @@ final class StandaloneApiServer(
     preloadPackages(packageStore)
 
     val owner = for {
-      initialConditions <- ResourceOwner.forFuture(() =>
-        readService.getLedgerInitialConditions().runWith(Sink.head))
-      authorizer = new Authorizer(
-        () => java.time.Clock.systemUTC.instant(),
-        initialConditions.ledgerId,
-        participantId)
       indexService <- JdbcIndex
         .owner(
           ServerRole.ApiServer,
-          initialConditions.config,
-          domain.LedgerId(initialConditions.ledgerId),
+          ledgerConfig.initialConfiguration,
```

Comment:
This is a breaking change:
- the ledger ID is not given by the ledger backend (ReadService) anymore
- the initial time model is not given by the ledger backend (ReadService) anymore

I think the part about the time model should be added to the changelog as well.

Reply:
I think we should actually rather try to fetch the configuration from the database. We have the infrastructure now in place. Both here and in `LedgerBackedIndexServer#getLedgerConfiguration`. What do you think?

Reply:
Yes, `JdbcIndex` should not depend on the initial configuration.

The only component that should access the initial configuration is `LedgerConfigProvider`, which is responsible for writing the initial configuration to the ledger if there is none.